src/components/Login.tsx

```diff
@@ -9,49 +9,107 @@ import {
   DialogActions,
   Button,
   TextField,
-  FormControl
-} from "@material-ui/core";
-import ExitToAppIcon from "@material-ui/icons/ExitToApp";
-import { AirDBContext } from "../context/AirDBContext";
+  FormControl,
+  useTheme
+} from '@material-ui/core'
+import { ExitToApp, Edit } from '@material-ui/icons'
+import Question from './take-test/Question'
+import { AirDBContext } from '../context/AirDBContext'
 
 export default React.memo((props: any) => {
-  const { users, updateAirDB } = React.useContext(AirDBContext);
-  const [toggled, setToggled] = React.useState(false);
-  const [username, setUser] = React.useState("");
-  const [password, setPassword] = React.useState("");
+  const { users, updateAirDB } = React.useContext(AirDBContext)
+  const [toggled, setToggled] = React.useState(false)
```

Comment:
In the future, changing all you're quotes to use single may make sense to be in a branch of its own so that changes and file count are easier to scan. Just a suggestion for the future.

Reply:
Is there a package or github action for that?

Reply:
Christian has mentioned it. How would I do that?